theano/tensor/nnet/nnet.py

```diff
@@ -29,6 +29,7 @@
 from theano.tensor.nnet.sigm import sigmoid, softplus
 from theano.gradient import DisconnectedType
 from theano.gradient import grad_not_implemented
+from theano.sandbox.blocksparse import sparse_block_dot
```

Comment:
@nouiz It seems to me that sparse_block_dot is general enough to be in ```tensor``` rather ```tensor.nnet```. The function ```h_softmax``` however should definitely be in ```tensor.nnet```. Do you have any objection?